src/components/ParkingEditor.jsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import {
+    Link
+} from 'react-router-dom';
+import axios from 'axios';
+import '../styles/ParkingEditor.css'
+import { CountryDropdown, RegionDropdown, CountryRegionData } from 'react-country-region-selector';
+import EventService from "../services/EventService";
+
+class ParkingEditor extends React.Component {
+    constructor(props) {
+        super(props);
+
+        const URL = `http://localhost:${process.env.PORT || 5000}`;
+        this.apiClient = axios.create({
```

Comment:
I think you can reuse the apiClient in EventService.js
That one comes with security header support, so you don't need to pass it.